src/types.ts

```diff
@@ -1,3 +1,20 @@
+export interface CreateEnvOptions {
+  /**
+   * Enable Node.js compatibility (nodeless) preset.
```

Comment:
Explicitly add the default value

Reply:
I think **Enable ...** implies that default is false won't it?

Reply:
https://github.com/unjs/unenv/pull/373/files#diff-c54113cf61ec99691748a3890bfbeb00e10efb3f0a76f03a0fd9ec49072e410aR5